docs/examples/tcp_LAN_WAN_transport.rst

```diff
@@ -17,20 +17,27 @@ TCP over LAN can be tested in a scenario where several machines share the same L
 nevertheless, UDP performs better and is the advisable choice in practical situations.
 For this configuration example, one of the machines set up eProsima Shapes Demo as a server and all the others as
 clients.
-Assume that the server LAN IP address is ``192.168.2.49``, then
+Assume that the server LAN IP address is ``192.168.1.75``, then
 all clients instances of eProsima Shapes Demo deployed on other machines must specify this IP Server address.
 In this case, the 5100 port is selected as TCP port, but any other available TCP port is valid.
 
 .. warning::
 
     The server firewall must allow inbound traffic on the selected port.
 
+Server TCP side:
 
-.. image:: /01-figures/LAN_settings_options.png
+.. image:: /01-figures/tcp_lan_server_options.png
    :scale: 100 %
-   :alt: LAN settings options
+   :alt: LAN Server settings options
    :align: center
 
+Client TCP side:
+
+.. image:: /01-figures/tcp_client_options.png
```

Comment:
I am completely stunned that you have been able to take this screenshot. When the `TCP LAN Server` option is selected, only the `Listen Port` option should be enabled. :thinking: Clearly here the `TCP Client` option should be appearing in the drop down list.

Reply:
_It is not a bug, it is a feature :+1:_